android/KMEA/app/src/main/java/com/tavultesoft/kmea/packages/PackageProcessor.java

```diff
@@ -72,7 +72,8 @@ public PackageProcessor(File resourceRoot) {
   public String getPackageID(File path) {
     // Sometimes, downloading a kmp multiple times results in a filename with a suffix:
     // " (#).kmp" or "-#.kmp" in the filename, so strip out the number
-    String filename = path.getName().toLowerCase().replaceAll("(\\s+\\(|-)\\d+\\)?.kmp$", ".kmp");
+    String filename = path.getName().toLowerCase().replaceAll(
+      "\\s*((\\(\\d+\\))|(-\\d+))\\.kmp$", ".kmp");
```

Comment:
I don't think it's quite right yet. This will match `khmer_angkor -1.kmp`?

Reply:
I didn't know if other Android devices might have whitespace before `-1.kmp` so I wanted to strip that out.
"khmer_angkor  1.kmp" would then become "khmer_angkor.kmp"

Reply:
OK, but `khmer_angkor 1.kmp` wouldn't match, would it?

Reply:
True. I think we just update regex to handle "khmer_angkor 1.kmp" if we start seeing those in Sentry.